htdocs/install/mysql/migration/17.0.0-18.0.0.sql

```diff
@@ -82,4 +82,13 @@ ALTER TABLE llx_supplier_proposal ADD INDEX idx_supplier_proposal_fk_user_valid
 ALTER TABLE llx_supplier_proposal ADD INDEX idx_supplier_proposal_fk_projet (fk_projet);
 ALTER TABLE llx_supplier_proposal ADD INDEX idx_supplier_proposal_fk_account(fk_account);
 
+ALTER TABLE llx_facturedet ADD COLUMN situation_percent_cumulated real DEFAULT 100 AFTER fk_prev_id;
+ALTER TABLE llx_facturedet ADD COLUMN situation_total_ht_cumulated double(24,8) AFTER situation_percent_cumulated;
```

Comment:
We don't need this information.

We already have a chain between each situation invoice. The chain is build with existing fields
situation_cycle_ref    and    situation_counter

And the "cumulated amount" is the sum of each invoice into the same chain (with situation_counter lower). Adding a field to store is a duplication of info and will generate troubles.